common/setups/rasr/util/gmm.py

```diff
@@ -241,11 +244,11 @@ class PrevCtm:
     Defines a previous ctm output from a recognition which is used to extract the
     speaker alignments for SAT adaptation for the dev/train sets.
 
-    :param prev_step_key: name of the step, e.g. "tri"
-    :param pronunciation_scale: one of the existing pronunciation scales
-    :param lm_scale: one of the existing lm-scales
-    :param iteration: one of the existing GMM iterations
-    :param optimized_lm: use automatically optimized lm.scale which might differ from param lm_scale
+    prev_step_key: name of the step, e.g. "tri"
```

Comment:
This was correct before, `dataclass` allows to set `:param` in the docstring.

Reply:
`:param prev_step_key:`

this gives me an `Unresolved reference` error in PyCharm

removing `:` is enough to fix this, removing `"param` is not necessary. Or is this incorrect from PyCharm?

Reply:
seems to be a problem of PyCharm: https://youtrack.jetbrains.com/issue/PY-28549

only a quick google search so might also be something else